.github/workflows/ci-publish-test-results.yml

```diff
@@ -0,0 +1,81 @@
+# This workflow publishes a summary of unit test results. This has to be done as a separate workflow because forks
+# don't have the right permissions.
+#
+# See https://github.com/EnricoMi/publish-unit-test-result-action#support-fork-repositories-and-dependabot-branches
+# for more.
+
+name: Publish unit test results
+
+on:
+   workflow_run:
+      workflows: ["ci-test"]
+      types:
+         - completed
+
+jobs:
+   unit-test-results:
+      name: Unit test results
+      runs-on: ubuntu-latest
+      if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
+      steps:
+         - name: Download artifacts
+           uses: actions/github-script@v3
```

Comment:
I thought I left a comment here about pulling this in to a separate action.     I'm pushing Young as well to not commit in large scripts like this in to the workflow files.   Might make sense to collab on a template or such.   I've only built actions with shell script, so not sure what all is involved, but we can't use actions in actions.  🤷

Reply:
Alternative, use the published action in market place (https://github.com/marketplace/actions/publish-unit-test-results) or publish our own.

Reply:
> Alternative, use the published action in market place (github.com/marketplace/actions/publish-unit-test-results) or publish our own.

We are using our fork of the action for the main logic, to be clear -- this is a preparatory step before the action can be used.

Reply:
> I thought I left a comment here about pulling this in to a separate action.

Would putting it in a separate file be OK? Seems like we can do that much.

Reply:
Moved it to a separate file and renamed it to `ci-test-complete`.

Reply:
Per discussion with @rexhoffman, moved the action to https://github.com/diem/actions/tree/master/download-artifacts.